app/src/main/AndroidManifest.xml

```diff
@@ -38,7 +38,8 @@
             android:name=".activities.SignInAsGuestActivity"
             android:theme="@style/AppTheme.NoActionBar" />
         <activity android:name=".activities.UsersActivity" />
-        <activity android:name=".activities.ProfileActivity" />
+        <activity android:name=".activities.ProfileActivity"
+            android:theme="@style/SignInTheme" />
```

Comment:
even if this might be correct from a functional point of view ... SignInTheme is not an appropriate naming for the theme anymore. Please find a better wording 😄